docs/modules/clients/pages/java.adoc

```diff
@@ -1715,205 +1447,140 @@ YAML::
-to connect to alternative clusters, the current <<client-network, member list>> is not considered;
-the client cuts all the connections before attempting to connect to a new cluster and tries the clusters as configured.
-See the below configuration related sections.
+[source,java]
+----
+SocketOptions socketOptions = clientConfig.getNetworkConfig().getSocketOptions();
+socketOptions.setBufferSize(32)
+             .setKeepAlive(true)
+             .setTcpNoDelay(true)
+             .setReuseAddress(true)
+             .setLingerSeconds(3);
+----
 
-=== Ordering of Clusters When Clients Try to Connect
+=== Enable client TLS/SSL
 
-The order of the clusters, that the client will try to connect
-in a blue-green or disaster recovery scenario, is decided by
-the order of these cluster declarations as given in the client configuration.
+[blue]*Hazelcast {enterprise-product-name}*
 
-Each time the client is disconnected from a cluster and it cannot connect back to the same one,
-the configured list is iterated over. Count of these iterations before
-the client decides to shut down is provided using the `try-count` configuration element.
-See the following configuration related sections.
+You can use TLS/SSL to secure the connection between the client and the members.
+If you want TLS/SSL enabled for the client-cluster connection, you should set `SSLConfig`.
+Once set, the connection (socket) is established out of an TLS/SSL factory defined either by
+a factory class name or factory implementation. Fore more information, see xref:security:tls-ssl.adoc[TLS/SSL].
 
-We didn't go over the configuration yet (see the following configuration related sections),
-but for the sake of explaining the ordering, assume that you have
-`client-config1`, `client-config2` and `client-config3`
-in the given order as shown below (in your `hazelcast-client-failover` XML or YAML file).
-This means you have three alternative clusters.
+As explained in the TLS/SSL section, Hazelcast members use keyStores to
+identify themselves (to other members) and Hazelcast clients use trustStores to
+define which members they can trust. The clients also have their own keyStores, and
+members have their own trustStores, so that members can
+know which clients they can trust. For more information, see xref:security:tls-ssl.adoc#mutual-authentication[Mutual authentication].
 
-[tabs] 
-==== 
-XML:: 
-+ 
--- 
-[source,xml]
-----
-<hazelcast-client-failover>
-    <try-count>4</try-count>
-    <clients>
-        <client>client-config1.xml</client>
-        <client>client-config2.xml</client>
-        <client>client-config3.xml</client>
-    </clients>
-</hazelcast-client-failover>
-----
---
-
-YAML::
-+
-[source,yaml]
-----
-hazelcast-client-failover:
-  try-count: 4
-  clients:
-    - client-config1.yaml
-    - client-config2.yaml
-    - client-config3.yaml
-----
-====
+// check standalone - is config different or is the content pulled from above linked page?
```

Comment:
the ssl/tls config should be the same 🤔

Reply:
I highlighted this one as it looks like @sergey-serdyukov PR has the changes compared to our original page - please confirm!

Reply:
Confirmed. ssl/tls settings are identical in both clients

Reply:
I think the issue is that in the original PR the Cloud section is removed but the TLS code samples from this section have been kept and moved under TLS section: https://github.com/hazelcast/hz-docs/pull/1240/files#r1761451399

Maybe the code sample works for both cases, will need some guidance here but I could link from here to the programmatic sample for the standalone client?

Reply:
Implemented above suggestion - reopen comment if you have alt suggestion